dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -158,6 +158,18 @@ DataInterfaceIIDM::dumpToFile(const std::string& iidmFilePath) const {
   }
 }
 
+void
+DataInterfaceIIDM::dumpToFile(std::ostream& stream) const {
+  try {
+    stdcxx::Properties properties;
+    powsybl::iidm::converter::ExportOptions options(properties);
+
+    powsybl::iidm::Network::writeXml("", stream, *networkIIDM_, options);
```

Comment:
What is initially the usage for the first parameter?

Reply:
It's to generate a file which contains data to cipher the data in outputIIDM.xml (I guess). It's useless in our use case because we have to set the properties to use this facility like this :

```
stdcxx::Properties properties;
properties.set("iidm.export.xml.anonymised", "true");
```